### PR TITLE
Update deps to latest.

### PR DESCRIPTION
**Pull request by JSAssassin**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -67,11 +43,38 @@ jobs:
...
-        npm run coverage-ci
+        npm test
       env:
         CI: true
-    - name: Upload coverage to Codecov
-      uses: codecov/codecov-action@v1
-      with:
-        file: ./test/coverage/lcov.info
-        fail_ci_if_error: true
+  # coverage:
```
Comment (mattcollier) [inline .github/workflows/main.yml:49]: Please add comment here that coverage is being disabled in connection with this:
https://github.com/digitalbazaar/bedrock-karma/issues/14

**Review by mattcollier — APPROVED** [review]
TY

File: .github/workflows/main.yml (hunk 1 above)
Comment (JSAssassin) [reply .github/workflows/main.yml:55]: Added ef6f641.

**Review by aljones15 — APPROVED** [review]
Good work here and thank you. All deps still install and the test project runs (there are no tests though!). Not sure on what's going on with this PR, but you have 2 appovals so feel free to merge.

**Pull request merged**